src/ast/semantic_analyser.h

```diff
@@ -100,7 +100,13 @@ class SemanticAnalyser : public Visitor {
   void accept_statements(StatementList *stmts);
 
   Probe *probe_;
+
+  // Temporarily record the function currently being visited by this SemanticAnalyser.
```

Comment:
Wording nit. Same for below
```suggestion
  // Holds the function currently being visited
```

Reply:
Done.